plugins/ftdidmx/README.md

```diff
@@ -1,10 +1,34 @@
-FTDI USB Chipset DMX Plugin
-===========================
```

Comment:
Can we keep the ==== on the title to match all the other plugins please.

Reply:
OK